packages/edit-post/src/components/preferences-modal/meta-boxes-section.js

```diff
@@ -23,8 +23,7 @@ export function MetaBoxesSection( {
 	...sectionProps
 } ) {
 	// The 'Custom Fields' meta box is a special case that we handle separately.
-	const thirdPartyMetaBoxes = filter(
-		metaBoxes,
+	const thirdPartyMetaBoxes = metaBoxes.filter(
```

Comment:
`metaBoxes` will always be an array:

https://github.com/WordPress/gutenberg/blob/d8edb10b00f4aa2e3fa62327dbcca749557c4d5d/packages/edit-post/src/store/selectors.js#L410